src/dropdown/bs-dropdown.config.ts

```diff
@@ -11,4 +11,6 @@ export class BsDropdownConfig {
   insideClick = false;
   /** turn on/off animation */
   isAnimated = false;
+  /** value true of preventDefaultBehaviour allows event stopPropagation*/
+  preventDefaultBehaviour = false;
```

Comment:
naming is too wide,
please be more specific, for example 
`stopOnClickPropogation`